data/devices/marsgaming-mm4.device

```diff
@@ -0,0 +1,4 @@
+[Device]
```

Comment:
Add `DeviceType=mouse` somewhere here, otherwise libratbag will ignore this device file (since #931). This also causes test failures.